site/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>amb</title>
+  <link href="../assets/styles/normalize.css" rel="stylesheet" >
+  <link href="../assets/styles/main.css" rel="stylesheet" >
+</head>
+
+<body>
+  <header>
+    <a href="index.html">
+      <div>
+        <h3>amb</h3>
+      </div>
+    </a>
+
+    <nav>
+      <ul>
+        <li><a href="portfolio.html"> porotfolio </a></li>
+        <li><a href="blog.html"> blog </a> </li>
+        <li><a href="about.html"> about </a></li>
+        <li><a href="#contact"> contact </a> </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section id = "index_hero" class = "hero">
+      <h1>amb</h1>
+    </section>
+
+    <section class = "index_section1 section1">
+      <a href="portfolio.html">
+        <article class = "index_article">
+          <div>
+            <h3>portfolio</h3>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+            </p>
+          </div>
+        </article>
+      </a>
+
+      <a href="blog.html">
+        <article class = "index_article">
+          <div>
+            <h3>blog</h3>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+            </p>
+          </div>
+
+        </article>
+      </a>
+
+      <a href="about.html">
+        <article class = "index_article">
```

Comment:
Instead of using this class multiple times, you could give your main the class of "index" then do something like `.index article` in your css to target all of the articles on the index page. Same idea for your blog_article class and your portfolio_article class...